omnigan/trainer.py

```diff
@@ -681,6 +681,13 @@ def train(self):
 
             if self.logger.epoch % self.opts.train.save_n_epochs == 0:
                 self.save()
+            if (
+                self.logger.epoch >= self.opts.train.save_barrier
+                and (self.logger.epoch - self.opts.train.save_barrier)
+                % self.opts.train.save_frequency
+                == 0
+            ):
+                self.save(save_path_name="epoch_{}_ckpt.pth".format(self.logger.epoch))
```

Comment:
This logic can be handled inside save() as `self.logger.epoch` is available there